libs/base/ui/navigation/navigation-item/src/navigation-item.styles.ts

```diff
@@ -42,6 +49,24 @@ export const navigationItemStyles = css`
     border-radius: 4px;
   }
 
+  :host([divider])::after {
+    content: '';
+    background-color: var(--oryx-color-neutral-5);
+    position: absolute;
+    width: 80%;
+    height: 1px;
+    transform: translateY(var(--oryx-navigation-item-divider-position));
+  }
+
+  :host([divider]) {
+    margin-block-start: calc(var(--oryx-navigation-item-vertical-padding) * 2);
+  }
+
+  :host([divider][parent-collapsed])::after {
+    transform: translateY(var(--oryx-navigation-item-divider-position))
+      translateX(-33%);
+  }
```

Comment:
Seems overcomplicated, but I might miss something. I've simplified to:

```css

  :host([divider])::before {
    content: '';
    background-color: var(--oryx-color-neutral-5);
    width: calc(100% - 32px);
    height: 1px;
    position: absolute;
    inset-inline-start: 16px;
    margin-top: -64px;
  }
```

With this, it's not a breaking change and we do not need the MutationObserver

Reply:
Yes, using the MutationObserver was overcomplicating what was simple, thks